pmlmodelling/ersem
# Move support docs to readme

#### Description of Work

Fixes #91 

Moves the support documentation from a tab in read the docs documentation to the README.

#### Testing Instructions

1. Check the README correctly renders in the [docs](https://ersem--92.org.readthedocs.build/en/92/)
2. Check the wording is correct
